lib/bolt/cli.rb

```diff
@@ -218,8 +218,7 @@ def finalize_setup
 
         return unless !stdout.empty? && stdout.to_i < 3
 
-        msg = "Detected PowerShell 2 on controller. PowerShell 2 is deprecated and "\
-              "support will be removed in Bolt 3.0."
+        msg = "Detected PowerShell 2 on controller. PowerShell 2 is unsupported."
         Bolt::Logger.deprecation_warning("PowerShell 2 controller", msg)
```

Comment:
Do we want to just raise an error here instead of warning? Or would it be better to just see if Bolt happens to succeed, and let it fail on it's own if it fails?

Reply:
PowerShell 2 should only cause failures if you use the local transport, no? So failing outright just because you're in a PowerShell 2 environment seems wrong.

Reply:
In practice it only 'fails' if we are attempting a task or plan. Those two return json data, and we use a set of custom powershell code to do what `ConvertFrom-Json` handles natively in v3+. Other parts work since we just execute a string and return what it returns as a string.